extension/scripts/global/globalClasses.js

```diff
@@ -65,9 +65,9 @@ class FeatureManager {
 			})
 		);
 
-		document.find(".tt-page-status-header").onclick = function () {
-			this.classList.toggle("collapsed");
-			ttStorage.change({ filters: { containers: { [this.containerID]: this.classList.contains("collapsed") } } });
+		document.find(".tt-page-status-header").onclick = () => {
+			document.find(".tt-page-status-header").classList.toggle("collapsed");
+			ttStorage.change({ filters: { containers: { [this.containerID]: document.find(".tt-page-status-header").classList.contains("collapsed") } } });
```

Comment:
`classList.toggle` returns a boolean, to show whether or nor is now has the class or not, you can use this for the collapsed or not, instead of having to find the element again.

Reply:
Oh, thanks! Wasn't aware it returned anything.

Reply:
I didn't even know it existed at first xD